etc/docker/elyra/Dockerfile

```diff
@@ -39,9 +39,11 @@ RUN python -m pip install --upgrade pip && \
     npm install -g yarn && \
     npm install -g npm && \
     cd /tmp && git clone https://github.com/elyra-ai/elyra.git && \
-    cd /tmp/elyra && git checkout tags/v1.0.0rc3 -b v1.0.0rc3 && make install
+    cd /tmp/elyra && git checkout tags/v1.0.0rc2 -b v1.0.0rc2 && make install
 
 WORKDIR /home/jovyan/work
 
-CMD ["/usr/local/bin/start-elyra.sh"]
```

Comment:
this would break the integration with JupyterHub and JEG

Reply:
let me fix this

Reply:
What is JEG? I've changed from jupyter-lab to jupyterhub, but I can't run jupyter-lab from there, only jupyter notebooks...